### PR TITLE
normalize exact words

### PR DESCRIPTION
**Pull request by MarinPostma**
Normalize the exact words, as specified in the specification.

### CODE REVIEW

File: milli/src/update/settings.rs [hunk 1]
```diff
@@ -580,6 +580,23 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
     fn update_exact_words(&mut self) -> Result<()> {
         match self.exact_words {
             Setting::Set(ref mut words) => {
+                fn normalize(analyzer: &Analyzer<&[u8]>, text: &str) -> String {
+                    analyzer.analyze(text).tokens().map(|token| token.text().to_string()).collect()
```
Comment (Kerollmops) [inline milli/src/update/settings.rs:584]: Ok, so as I understand this function you are using the tokenizer to split and normalize the word for you, concatenating all of the normalized words together if there is more than one word. Is it expected?

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Could you fix the commit messages, please?

File: milli/src/update/settings.rs (hunk 1 above)
Comment (MarinPostma) [reply milli/src/update/settings.rs:584]: yes, for now @gmourier think it's better to ignore this edge case

**Review by Kerollmops — APPROVED** [review]
So if @gmourier is ok with that, then I approve those changes!

**Comment by curquiza** [comment]
bors merge

**Pull request merged**